### PR TITLE
Changes for uploading documents for v3 endpoint

### PR DESCRIPTION
**Pull request by ramahalingam**

### CODE REVIEW

File: tests/Hyperwallet/Tests/HyperwalletTest.php [hunk 1]
```diff
@@ -3535,6 +3535,56 @@ private function createAndInjectApiClientMock(Hyperwallet $client) {
...
+        $options = array(
+            'multipart' => [
+                [
+                    'name'     => 'data',
+                    'contents' => '{"documents":[{"type":"DRIVERS_LICENSE","country":"AL","category":"IDENTIFICATION"}]}'
+                ],
+                [
+                    'name'     => 'drivers_license_front',
+                    'contents' => fopen('/Users/ramahalingam/Desktop/L1.png', "r")
+                ],
```
Comment (arao6) [inline tests/Hyperwallet/Tests/HyperwalletTest.php:3568]: This looks like a path on your machine. Can we change/remove this?

File: tests/Hyperwallet/Tests/HyperwalletTest.php [hunk 2]
```diff
@@ -3535,6 +3535,56 @@ private function createAndInjectApiClientMock(Hyperwallet $client) {
...
+                [
+                    'name'     => 'drivers_license_back',
+                    'contents' => fopen('/Users/ramahalingam/Desktop/L2.png', 'r')
+                ]
+            ]
+        );
```
Comment (arao6) [inline tests/Hyperwallet/Tests/HyperwalletTest.php:3574]: Same as above

File: tests/Hyperwallet/Tests/HyperwalletTest.php (hunk 1 above)
Comment (Rrathinasabapath) [reply tests/Hyperwallet/Tests/HyperwalletTest.php:3568]: Please put the png file in a relative path within the test folder.

File: tests/Hyperwallet/Tests/HyperwalletTest.php (hunk 2 above)
Comment (Rrathinasabapath) [reply tests/Hyperwallet/Tests/HyperwalletTest.php:3574]: Please put the png file in a relative path within the test folder.

**Review by Rrathinasabapath — COMMENTED** [review]
Added comments about the png files' path. Pls fix this and let me know.

File: tests/Hyperwallet/Tests/HyperwalletTest.php (hunk 1 above)
Comment (ramahalingam) [reply tests/Hyperwallet/Tests/HyperwalletTest.php:3568]: Added the png files to the resources folder and changed the test accordingly.

File: tests/Hyperwallet/Tests/HyperwalletTest.php (hunk 2 above)
Comment (ramahalingam) [reply tests/Hyperwallet/Tests/HyperwalletTest.php:3574]: Added the png files to the resources folder and changed the test accordingly.

Comment (Rrathinasabapath) [reply tests/Hyperwallet/Tests/HyperwalletTest.php:3574]: Thanks for the changes

**Review by Rrathinasabapath — APPROVED** [review]
Reviewed the changes. Ready for a review by Peter.

**Review by peter-joseph — APPROVED** [review]

**Pull request merged**